hydra-cluster/test/Test/DirectChainSpec.hs

```diff
@@ -103,14 +103,16 @@ spec = around showLogsOnFailure $ do
         withIOManager $ \iocp -> do
           seedFromFaucet_ node aliceCardanoVk 100_000_000 Fuel (contramap FromFaucet tracer)
```

Comment:
these now need to be reverted since we are calling the faucet twice.

Reply:
yes maybe, just wanted to apply the minimum changes for this to pass ✅

Reply:
This is a different seedinf though (fuel vs. normal)

Reply:
aha yeah I didn't see it at first. Now we need to figure out why this fixes things.